app/controllers/home_controller.rb

```diff
@@ -1,12 +1,12 @@
 class HomeController < ApplicationController
 
   def index
-  @chapters = Chapter.all
-  @chapter = Chapter.all
+  @chapters = Chapter.where("is_active = 1")
```

Comment:
I recommend using [scopes](http://guides.rubyonrails.org/active_record_querying.html#scopes) in this case! They're super awesome.

In the `Chapter` model file, you would add something like this:

```
scope :active, -> { where(is_active: "1") }
```

Or if you change the `is_active` field to a boolean then

```
scope :active -> { where(is_active: true) }
```

Then throughout the code you can do this:

```
# Replace this
Chapter.where("is_active = 1")

# with this
Chapter.active
```

Scopes return an active record relation that allow you to chain queries nicely. They are the bomb.